src/cljc/quil/core.cljc

```diff
@@ -4963,3 +4963,20 @@
      [orig-name]
      (let [res (docs/matching-processing-methods fn-metas orig-name)]
        (u/print-definition-list res))))
+
+
+#?(:cljs 
+    (defn preload-font
+      "Fonts can be preloaded from URL, such as \"Arial.ttf\".
+      Be careful! Font name must be the same in preload-font call and in create-font call.
+
+      Sample:
+      (q/preload-font \"ComicSans.ttf\")
+      (q/text-font 
+        (q/create-font \"ComicSans.ttf\" 14))
```

Comment:
There is no `preload-font` in processing.js anyway, so it's not increasing gap. Quil is not a low level library so it's better if we hide these implementation details from users. If you user calls `create-font` from inside `:draw` then only the first call to preloading will have effect and all others will be cached so there won't be performance issues. 

But what happens if user tries to preload built-in font? Is it possible to know if user is using built-in font or font from  a file?